### PR TITLE
search bar added

### PR DESCRIPTION
**Pull request by AlNeon**
- [DEMO LINK](https://AlNeon.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://AlNeon.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,33 @@
...
+      method="post"
+      class="search__form"
+    >
+      <input
+        class="search__form__input big__input"
+        data-qa="keypress"
+        type="text"
+        name="big"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
```
Comment (VitaliyBondarenko1982) [inline src/index.html:26]: each input shuould have a label. 
wrap input in label.

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,33 @@
...
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      data-qa="big"
+      method="post"
+      class="search__form"
+    >
+      <input
+        class="search__form__input big__input"
```
Comment (VitaliyBondarenko1982) [inline src/index.html:21]:
```suggestion
        class="search__form__input"
```
pay attention on this point from [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
Screenshot from 2023-10-01 17-16-12

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Good job.
Need some improvements.

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+.search__form {
+  margin-top: 20px;
+}
+
+.search__form__input {
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```
Comment (VitaliyBodnar) [inline src/style.css:25]: 61, 78, 97 duplicated color, so make sense to move it to the variables
check other places

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

**Review by loralevitska — APPROVED** [review]
Well done)

**Pull request closed without merging**